netty/src/test/java/io/grpc/netty/NettyTestUtil.java

```diff
@@ -56,6 +57,13 @@ static ByteBuf compressionFrame(byte[] data) {
     return buf;
   }
 
+  // Buffer safe release: release until it's free.
```

Comment:
No, please no. This is asking to be misused, and I don't think it is appropriate even in the test. I do think we should clean up the buffers, but if the reference counting is wrong the test should fail.